docs/buildGlibcWASM.md

```diff
@@ -122,17 +127,62 @@ In the build directory, usually we use `make --keep-going -j$(nproc)`. The first
 This procedure is specified in the `gen_sysroot.sh` script in our glibc repo. It's main job is to generate a WASM sysroot structre like
 
 ```
-sysroot/
```

Comment:
This is the structure of WASM sysroot and should keep this part

Reply:
solved